owa/app/js/components/manage_patient_identifier_sources/edit_patient_identifier_sources/editPatientIdentifierSources.css

```diff
@@ -0,0 +1,12 @@
+.requiredField{
+    color: red;
+}
+
+.editPatientIdentifierSourcesInput{
```

Comment:
I'm sure the project ain't following modular CSS conventions but just for the future, CSS class names should be preferably be separated by a dash instead of camel case. the following links have a better explanation:
http://thesassway.com/advanced/modular-css-naming-conventions
https://css-tricks.com/bem-101/ - I personally prefer BEM for my class names it's more readable and also increases the rendering performance by reducing the amount of time required for style recalculations.

Reply:
@brayoh Noted. thank you.

Reply:
Nice observation @brayoh 👍